can't build in windows

I can't build in windows any longer. Seems like it's coming from the tauri-runtime-wry dependency.
```
   Compiling tauri-runtime v0.3.4
   Compiling tauri-runtime-wry v0.3.5
error[E0061]: this function takes 4 arguments but 3 arguments were supplied
   --> src\run_external.rs:128:44
    |
128 |                     let mut process = self.create_process(&input, true, head)?;
    |                                            ^^^^^^^^^^^^^^ ------  ----  ---- supplied 3 arguments
    |                                            |
    |                                            expected 4 arguments
    |
note: associated function defined here
   --> src\run_external.rs:332:8
    |
332 |     fn create_process(
    |        ^^^^^^^^^^^^^^
333 |         &self,
    |         -----
334 |         input: &PipelineData,
    |         --------------------
335 |         use_cmd: bool,
    |         -------------
336 |         use_external_terminal: bool,
    |         ---------------------------
337 |         span: Span,
    |         ----------

For more information about this error, try `rustc --explain E0061`.
error: could not compile `nana` due to previous error
Terminate batch job (Y/N)? Terminate batch job (Y/N)? y
```

--- a/src-tauri/src/run_external.rs
+++ b/src-tauri/src/run_external.rs
@@ -125,7 +125,7 @@ impl ExternalCommand {
         {
             match process.spawn() {
                 Err(_) => {
-                    let mut process = self.create_process(&input, true, head)?;
+                    let mut process = self.create_process(&input, true, false, head)?;
                     child = process.spawn();
                 }
                 Ok(process) => {

